doc/classes/Shape2D.xml

```diff
@@ -65,7 +65,7 @@
 	</methods>
 	<members>
 		<member name="custom_solver_bias" type="float" setter="set_custom_solver_bias" getter="get_custom_solver_bias" default="0.0">
-			The shape's custom solver bias.
+			The shape's custom solver bias. A solver bias is a factor controlling how much two objects "rebound", after violating a constraint, to avoid leaving them in that state because of numerical imprecision.
```

Comment:
I suppose this bias is in pixels in 2D, so the unit should be mentioned:

```suggestion
			The shape's custom solver bias (in pixels). A solver bias is a factor controlling how much two objects "rebound" after violating a constraint. This bias avoids leaving the objects in an constraint-violating state because of numerical imprecision.
```

Reply:
> I suppose this bias is in pixels in 2D, so the unit should be mentioned.

No, bias goes from 0.0 to 1.0.

Here is how it influences the bias if two shapes collide and one has a custom bias > 0: 
https://github.com/godotengine/godot/blob/0cee8831b25377567bff8f46f7eaad8cb203dbf8/servers/physics_2d/body_pair_2d_sw.cpp#L354-L367

I'm not sure how to word this for docs.